### PR TITLE
Test/self hosted runners

### PR DESCRIPTION
**Pull request by emalinowski**
### New Features
Created a self hosted runner workflow that builds on amd/arm nodes natively so that we can have arm images run without the need for emulation through qemu/binfmt. The runners are on dev2 and need to be setup manually for each repo.

### CODE REVIEW

File: .github/workflows/self_hosted_build_push.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            quay.io/cdis/${{ env.REPO_NAME }}:${{ env.IMAGE_TAG }}
+            ${{ inputs.AWS_ECR_REGISTRY }}/gen3/${{ env.REPO_NAME }}:${{ env.IMAGE_TAG }}
+
+      - name: Build and push ECR
+        if: ${{ !inputs.USE_QUAY_ONLY }}
+        uses: docker/build-push-action@v3
```
Comment (cmlsn) [inline .github/workflows/self_hosted_build_push.yaml:131]: Bump docker/build-push-action@v3  to **docker/build-push-action@v4**

**Review by cmlsn — CHANGES_REQUESTED** [review]
I would try and bump to v4 of the builder otherwise this looks good.

File: .github/workflows/self_hosted_build_push.yaml (hunk 1 above)
Comment (emalinowski) [reply .github/workflows/self_hosted_build_push.yaml]: Done

**Comment by emalinowski** [comment]
https://ctds-planx.atlassian.net/browse/GPE-711

**Pull request merged**